Src/Plugins/AzureDevOpsConfiguration/AZC101.BuildDefinitionSecurity.json

```diff
@@ -1,6 +1,8 @@
 {
-  "SharedStringsFileName": "AzureDevOpsConfiguration.SharedStrings.txt",
   "ValidatorsAssemblyName": "AzureDevOpsConfiguration.dll",
+  "SharedStringsFileName": "AzureDevOpsConfiguration.SharedStrings.txt",
+  "ExtensionName": "SecureAzureDevOps",
```

Comment:
>ExtensionName [](http://example.com/codeflow?start=3&length=13)

Every extension now has an explicit name, which is persisted to the associated tool component object.